lib/child_process.js

```diff
@@ -50,7 +50,24 @@ exports.fork = function(modulePath /*, args, options*/) {
 
   args = execArgv.concat([modulePath], args);
 
-  if (!Array.isArray(options.stdio)) {
+  if (typeof options.stdio == 'string') {
+    const stdioStringToArray = (option) => {
```

Comment:
While it doesn't depend on any variables in the `fork()` scope I did think it logically belonged inside of fork because it wasn't generic enough to be used by any of the other exported functions needing the `stdioStringToArray` operation.

Case in point, fork is the only function that requires `'ipc'` be append to the returned array. This is just my opinion though so comments/feedback are welcomed.